### PR TITLE
Strengthen T202.

### PR DESCRIPTION
**Pull request by Almanzoris**

### CODE REVIEW

**Comment by Moniker1998** [comment]
I'm not sure if this is correct:

https://topology.pi-base.org/spaces?q=locally+pseudocompact+%2B+paracompact+%2B+regular+%2B+not+pseudocompact

**Review by Moniker1998 — APPROVED** [review]
Everything looks correct. Removing the theorem that locally metrizable + Haudorff => metrizable will not cause any issues either. It can be deduced from this theorem along with other theorems. 

Note the added theorem can't be deduced from current known theorems.

**Comment by prabau** [comment]
Since this has not been merged yet, there was one thing I wanted to remark on.  Before, it was clear that the theorem was exactly the Smirnov metrization theorem.  Now it's not clear what is what anymore.  Let me propose a change below, also improving the reference.

File: theorems/T000202.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+  P000121: true
 refs:
 - zb: "1052.54001"
   name: General Topology (Willard)
 ---
 
-The "Smirnov metrization theorem" asserted in 23G.3 of {{zb:1052.54001}}.
+If $X$ is {P144}, {P134} and {P30}, its Kolmogorov quotient is {P82}, {P3} and {P30}, hence {P53}. Therefore, $X$ is {P121}.
+
+See the "Smirnov metrization theorem" asserted in 23G.3 of {{zb:1052.54001}}.
```
Comment (prabau) [inline theorems/T000202.md:17]:
```suggestion
If $X$ is {P144}, {P134} and {P30}, its Kolmogorov quotient is {P82}, {P3} and {P30}.
By the "Smirnov metrization theorem" (Theorem 42.1 in {{zb:0951.54001}}), a space with the last three properties is {P53}.
Hence $X$ is {P121}.
```

**Comment by prabau** [comment]
And then replace Willard with Munkres in the `refs:`.

**Pull request merged**